x-pack/plugins/security_solution/public/common/components/threat_match/index.tsx

```diff
@@ -5,18 +5,15 @@
  * 2.0.
  */
 
```

Comment:
This bug is reproducible in `main` with the only difference that unknown in the current context field names aren't visible. It become much more relevant in case of editing prebuilt rules. Users without ingested data should be able to save the rule and mapping fields shouldn't disappear.

Taking above into account I fixed this bug in the last commit by adding a non blocking validation for threat match mapping field. It works in a similar way EQL/ESQL query fields validation does. PR's description has been updated accordingly.